README.md

```diff
@@ -4,4 +4,25 @@ Version
 Run 
 - 'npm i' to install dependencies
 - 'npm run index' to generate output 
-- 'npm run test' to run test cases
\ No newline at end of file
+- 'npm run test' to run test cases
+
+## Specs
+
+### League Rules
+Rankings are determined by points. Points are awarded to teams as follows:
+- A win is worth 3 points
+- A draw is worth 1 point
+- A loss is worth 0 points
+
+### Input
+The expected input is a text file (see `input.txt` for valid example input)
+
+### Output
+The expected output is a text file. Each time the software reaches the end of a complete match day, it should output the results of that match day. A complete match day consists of three games.

```

Comment:
note that `A complete match day consists of three games.` gives the answer to one of the bugs, but this implementation really doesn't conform to the takehome spec where the end of a matchday is determined by checking whether each team has played once